src/pages/deriv-go/components/_dgo-hero.js

```diff
@@ -0,0 +1,315 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import { localize } from 'components/localization'
+import { Flex } from 'components/containers'
+import { Header, QueryImage } from 'components/elements'
+import { LinkButton } from 'components/form'
+import device from 'themes/device.js'
+import Dgoogle_play_Logo from '/home/deriv/Desktop/deriv-com/src/images/common/derivgo/derivgo_gogleplay.png'
```

Comment:
This asset seems to be breaking the build. Please include the `derivgo_gogleplay.png` image file as well